<?xml version="1.0"?>
<launch>

    <arg name="full_path_imu" default="/dev/microstrain/3dm_gx5_45_0000__6251.65901"/>

    <node pkg="zeabus_sensor" type="imu_node" name="sensor_imu" output="screen">

        <param name="full_path_port" type="string" value="$(arg full_path_imu)"/>

    </node>

</launch>
